controllers/secret_controller.go

```diff
@@ -138,8 +147,9 @@ type SecretReconciler struct {
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *SecretReconciler) Reconcile(ctx context.Context,
 	request ctrl.Request) (result reconcile.Result, reconcileErr error) {
-	log := r.log.WithValues(SecretController, request.NamespacedName)
+	_ = r.log.WithValues(SecretController, request.NamespacedName)
```

Comment:
I think this the left side should become `r.log` since WithValues returns a new logr

https://sdk.operatorframework.io/docs/building-operators/golang/references/logging/#creating-a-structured-log-statement